src/test/java/bio/terra/integration/DataRepoClient.java

```diff
@@ -159,7 +159,11 @@ public <T> DataRepoResponse<T> waitForResponse(
 
     if (jobModelResponse.getStatusCode() != HttpStatus.OK) {
       throw new IllegalStateException(
-          "unexpected job status code: " + jobModelResponse.getStatusCode());
+          "unexpected job status:"
+              + "\nwith code: "
+              + jobModelResponse.getStatusCode()
```

Comment:
Can this be done in `DataRepoResponse`? It doesn't appear to be specific to its use in this method. Looking at the code further, it could probably be done in `ObjectOrErrorResponse`.

Reply:
Actually, looking, not really.  This is really more contextual.  At least not for now (and I want to be careful about our logging: we're already pushing the limits beyond which gradle scans won't publish